projects/hslayers/src/components/layermanager/layermanager-metadata.service.ts

```diff
@@ -314,13 +314,14 @@ export class HsLayerManagerMetadataService {
       const layerNameInParams: string = params.LAYERS;
 
       this.parseWmsCaps(layerDescriptor, layerNameInParams, caps);
-      if (getSubLayers(layer)) {
-        /* When capabilities have been queried, it's safe to override LAYERS 
-         param now to not render the container layer, but sublayers. */
-        this.HsLayerUtilsService.updateLayerParams(layer, {
-          LAYERS: getSubLayers(layer),
-        });
-      }
+      //*NOTE THIS BREAKS COPYING AND LOADING FROM STORAGE OR COMPOSITION
```

Comment:
But we need it so sublayers would be shown on the map instead of the preconfigured container layer name.

Reply:
Can't we copy also the cached capabilities so it doesnt query them again? In what way does it break the loadign from storage?

Reply:
@raitisbe I agree, it is just that the LAYERS param looses the initial layer name set inside the params, and layer one it causes the weird behaviour which I already mentioned before.

Reply:
@raitisbe Agree, but now I cannot create identical copy of this layer, because its cachedCaps are now just ->